spec/music_album_spec.rb

```diff
@@ -0,0 +1,18 @@
+require_relative 'helper_spec'
+
```

Comment:
- [ ] OPTIONAL
Kindly ensure to add test for `can_be_archived?()` method in the `Item` class, which should 
  * override the method from the parent class
  * return true if parent's method returns true AND if on_spotify equals true
  * otherwise, it should return false